### PR TITLE
Compas trial wasting treatment doc

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]
I'm having trouble reviewing what has changed in this doc, but I'll approve it because the parts I can review look right.  Can Ali and Abie go through the key points together this week?

File: docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+
+  * - Restriction
+    - Value
+    - Note
+  * - Male only
+    - False
+    -
+  * - Female only
+    -
+    - False
```
Comment (SylLutze) [inline docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst:270]: Should be one line up to fall in the correct box in the table

File: docs/source/models/intervention_models/wasting_treatment/ethiopia_wasting_treatment/index.rst [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+
+**Treatment**
+
+Congratulate and counsel the mother on appropriate IYCF practices.
+
+.. todo::
+    What about MAM with oedema? Are they treated as SAM or MAM?
+    Answer: they are treated as MAM in GBD, but in reality, probably SAM. MAM with oedema is approximately 2% of MAM.
+
+:underline:`Classify no acute malnutritionin children`
```
Comment (SylLutze) [inline docs/source/models/intervention_models/wasting_treatment/ethiopia_wasting_treatment/index.rst:285]:
```suggestion
:underline:`Classify no acute malnutrition children`
```

File: docs/source/models/intervention_models/wasting_treatment/ethiopia_wasting_treatment/index.rst [hunk 3]
```diff
@@ -0,0 +1,869 @@
...
+      - 20.8
+      - point value
+      - NOTE: this study reports mean duration of therapy without specifying if this duration is conditional on recovery. It is currently being implemented as time to recovery in our model. Of note, 81% of participants were successfully discharged in this study.
+      - [Woeltje_2020]_; Malawi
+
+
+Affected Outcomes
++++++++++++++++++
+
+The Vivarium modeling strategy above details how to solve for the transition rates among the covered uncovered populations. However, the wasting treatment intervention will be implemented as a variable that affects the relative risk of certain transition rates between wasting states in the :ref:`dynamic wasting model <2020_risk_exposure_wasting_state_exposure>`. The following table details the relative risks for each dynamic wasting model transition rate that is affected by wasting treatment based on a given treatment category.
```
Comment (SylLutze) [inline docs/source/models/intervention_models/wasting_treatment/ethiopia_wasting_treatment/index.rst:586]: In the first line of this paragraph you say the "covered uncovered populations"

File: docs/source/models/intervention_models/wasting_treatment/ethiopia_wasting_treatment/index.rst [hunk 4]
```diff
@@ -0,0 +1,869 @@
...
+
+  * - Restriction
+    - Value
+    - Note
+  * - Male only
+    - False
+    -
+  * - Female only
+    -
+    - False
```
Comment (SylLutze) [inline docs/source/models/intervention_models/wasting_treatment/ethiopia_wasting_treatment/index.rst:678]: Should be one line up again

**Review by SylLutze — APPROVED** [review]

**Pull request merged**